### ISSUE TITLE
Tavern: chat tab isnt present

### ISSUE THREAD

**Issue by saraolson** [HabitRPG/habitrpg-ios#586]
this may relate to how the tavern app bar seems too tall, but i just noticed you're currently unable to switch tabs without swiping. Also if you do swipe, the messages view loads weird, like the area to type your message isnt fixed to the bottom so if you scroll you can see other messages peek through below it
![image uploaded from ios 5](https://user-images.githubusercontent.com/12453119/35773039-f088bf4c-0914-11e8-8747-a6c0343ffbb8.png)
